float32 thumb_pos
float32 index_pos
float32 middle_pos

float32 thumb_torque
float32 index_torque
float32 middle_torque
